text/000-sort-variable-unification.md

```diff
@@ -0,0 +1,77 @@
+- Title: Introducing quality variables in sort unification
+
+- Drivers: Pierre-Marie Pédrot
+
+----
+
+# Summary
+
+In addition to variables for universe levels, we generalize the type of sorts to
+also contain quality variables, allowing to delay the choice between Prop, Type
```

Comment:
Minor English grammar nit:
```suggestion
also contain quality variables, which allows delaying the choice between Prop, Type
```
or
```suggestion
also contain quality variables, allowing delaying the choice between Prop, Type
```
which reads a bit more awkwardly to me, but is more correct than "allowing to delay"